app/controllers/people_controller.rb

```diff
@@ -68,7 +71,7 @@ def inactive
   end
 
   def index
-    @people = Person.active.select{|person| person.department&.name == 'CERT' || person.department&.name == 'Police'}
+    @people = Person.active.select{|person| person.department&.shortname == 'CERT' || person.department&.shortname == 'BAUX'}
```

Comment:
I agree with @pcai that the query in this PR

1. is an "n + 1" query, which is non-ideal
1. filters by instantiating all People in Ruby, which is non-ideal

I like Peter's first version of the query best.  Whether the second version makes use of that index is beyond my intuition (thanks @pcai , I will look into it further!)

Another possibility is: since the view instantiates each user's department to get its name, maybe Peter's second query should be altered by the addition of `includes(:department)`, and would then perform best.

I think the real-world difference in expected performance among these suggestions is pretty small.  However, in the real world, they are *all* much more performant (and better practice) than the query in this PR.